sunkit_image/radial.py

```diff
@@ -360,75 +362,14 @@ def nrgf(
     return new_map
 
 
-def set_attenuation_coefficients(order, range_mean=None, range_std=None, cutoff=0):
```

Comment:
Can we just keep this private and have the function call to this?

I don't see the upside to adding this code directly to the main function.